### ISSUE TITLE
how to understand " Do not call this op with the   output of `softmax`, as it will produce incorrect results. "

### ISSUE THREAD

**Issue by showintime** [tensorflow/tensorflow#34986]
So why?
[tf.nn.softmax_cross_entropy_with_logits_v2](https://github.com/tensorflow/tensorflow/blob/r1.15/tensorflow/python/ops/nn_ops.py#L3108-L3224)

**Comment by ravikyram** [comment]
@showintime 

Can you please go through [link](https://stackoverflow.com/questions/34240703/what-is-logits-softmax-and-softmax-cross-entropy-with-logits) and see if it helps you. Thanks!

**Comment by showintime** [comment]
thanks a lot,and i learned it.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"ravikyram"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月11日(星期三) 下午2:23
收件人:&nbsp;"tensorflow/tensorflow"<tensorflow@noreply.github.com&gt;;
抄送:&nbsp;"记得刷通识理论选修"<974200701@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [tensorflow/tensorflow] how to understand " Do not call this op with the   output of `softmax`, as it will produce incorrect results. " (#34986)




@showintime
 
Can you please go through link and see if it helps you. Thanks!

**Comment by ravikyram** [comment]
@showintime 

I am closing this thread as it solves your question. Thanks!
